src/snowflake/cli/api/project/definition.py

```diff
@@ -58,15 +61,20 @@ def _get_merged_definitions(paths: List[Path]) -> Optional[Definition]:
 
 
 def load_project(
-    paths: List[Path], context_overrides: Optional[Context] = None
+    paths: List[Path],
+    context_overrides: Optional[Context] = None,
+    render_env_values: bool = True,
 ) -> ProjectProperties:
     """
     Loads project definition, optionally overriding values. Definition values
     are merged in left-to-right order (increasing precedence).
     Templating is also applied after the merging process.
     """
     merged_definitions = _get_merged_definitions(paths)
-    return render_definition_template(merged_definitions, context_overrides or {})
+    if render_env_values:
+        return render_definition_template(merged_definitions, context_overrides or {})
+    else:
+        return raw_project_properties(merged_definitions)
```

Comment:
@sfc-gh-melnacouzi WDYT?

Reply:
To add a bit of a context: this was added, as Tomasz pointed out we need to test if templated project files are migrated unchanged from v1 to v2. 
The default behaviour, while loading project file, was to resolve any enviroment variable references with values. This solution is aimed to allow `DefinitionManager` to load and return project definition without processing envs.
As the default value for `render_env_values` is set to True, there should be no change in the function behaviour if called the old way.

Reply:
It's not only env. Let's change the name of the boolean to not use "env". Maybe use "render_templates".